webapp/src/utils/jira_issue_metadata.tsx

```diff
@@ -414,11 +414,11 @@ export function generateJQLStringFromSubscriptionFilters(issueMetadata: IssueMet
         });
 
         if (inclusion === FilterFieldInclusion.INCLUDE_ALL && values.length > 1) {
-            const clauses = chosenValueLabels.map((v) => `${quoteGuard(fieldName)} IN (${quoteGuard(v)})`);
+            const clauses = chosenValueLabels.map((v) => `${quoteGuard(fieldName)} IN (${quoteGuard(v.toString())})`);
```

Comment:
I dont think there should be a need to use toString here

Reply:
We need to add "toString()" to resolve the below TS error
`Argument of type 'string | ReactElement<any, string | JSXElementConstructor<any>>' is not assignable to parameter of type 'string'.
  Type 'ReactElement<any, string | JSXElementConstructor<any>>' is not assignable to type 'string'.`